src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,370 @@
+choose, and we'd never wish to take that away from them. For most people, modding is a hobby -- there will never be a
+good reason to deny developers more choice, and an understanding of this issue is not an excuse to write off 
+developers' hard work.
+
+We all have the power to approach this issue in a mature way, and we owe it to the hard-working mod developers to do 
+that. Speaking of which, if you're a mod developer that would like to look into writing cross-loader mods, please feel 
+free to join us on Discord -- there are plenty of developers there that can point you in the right direction.
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}What's different about Quilt's governance?{% endsummary %}
+
+Most projects that you'll find on the Internet are governed in a top-down manner, as a form of strict hierarchy. This
+often results in a tree-like or pyramid-like structure, in which the people at the top can, in one way or another,
+decide the fate of the entire project, and exert influence on the people below them.
+
+By contrast, Quilt uses a far less centralised structure. This results in a separation of responsibilities and control,
+split between the teams that are best suited to deal with that particular part of the project. As of this writing,
+Quilt's governmental structure is split into three major categories:
+
+* The Admin Board
+* The Community Team
+* The Development Teams
+
+While everyone that's a part of Quilt works alongside each other on a daily basis, there is no enforced structure of
+superiority -- only a separation of responsibilities. Hierarchies exist where it's necessary and productive for them 
+to, but they exist as a mechanism to further divide areas of work and to help staff members to specialise where they
+need to, not to act as an explicit chain of internal authority.
+
+For a full listing of our teams and who's part of them, please see [the Team Listings page](/about/teams/).
+
+### The Admin Board
+
+The Admin Board exists to help steer the project's overall direction, and to represent the project in situations where
+it's not appropriate for the Community Team to do so -- as well as handling miscellaneous tasks that require familiarity
+and oversight over the rest of the project. Their responsibilities include:
+
+* Representing The Quilt Project in sponsorships and other types of partnership, other than community collaboration
+* Acting as a form of oversight over the rest of the project, ensuring that processes are followed as documented and
+  aiding with internal conflicts
+* Acting as tie-breakers in Quilt's internal democratic processes, voting on issues that are in stalemates to get them
+  moving along again
+* Managing The Quilt Project's finances, and
+  [ensuring that income and outgoings are documented transparently](https://opencollective.com/quiltmc)
```

Comment:
Linking to the OpenCollective page doesn't explain how that makes it transparently unless people know what OpenCollective is.

Reply:
I mean, wouldn't clicking the link and seeing the financial stuff directly on the page make it obvious?

Reply:
If you don't scroll, it just looks like fancy Patreon.

Reply:
Right, but I did literally just mention transparency in the link, I think someone interested in that would be likely to scroll down to find that info haha